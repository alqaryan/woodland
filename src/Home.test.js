import { Platform } from 'react-native';

it(`Platform.OS works as expected`, () => {
    expect(Platform.OS).toMatchSnapshot();
});
