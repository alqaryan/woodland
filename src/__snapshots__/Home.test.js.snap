// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Platform.OS works as expected 1`] = `"ios"`;
